cloud_governance/policy/common_policies/send_aggregated_alerts.py

```diff
@@ -30,7 +30,7 @@ def __get_es_data(self):
         :return:
         :rtype:
         """
-        current_date = (datetime.utcnow().date()).__str__()
+        current_date = (datetime.now(timezone.utc).date()).__str__()
```

Comment:
Can you search in the code if we have more places to fix it ?

Reply:
I'll open a separate PR on it. It showed me a warning sign on the page, so I changed it here.

Reply:
I think we should not wait, it should be part of upgrade to 4.12

Reply:
@athiruma , Pls open a PR for it ASAP !!
I will approve it this time.